content/get-started/new-leads/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Set up your own community
+---
+
+There are many communities established within Galaxy based on [**region**](/community/#regional-communities) or [**domain of science**](/community/#regional-communities) but if you feel like you just don't see your own or don't feel identified with them, you can create your own community!
```

Comment:
The link to domain specific communities is not correct (for now its point to regional communities).